server/fleet/hosts.go

```diff
@@ -1024,3 +1034,14 @@ type HostSoftwareInstalledPath struct {
 	// InstalledPath is the file system path where the software is installed
 	InstalledPath string `db:"installed_path"`
 }
+
+// HostMacOSProfile represents a macOS profile installed on a host as reported by the macos_profiles
+// table of the macadmins oquery extension
+type HostMacOSProfile struct {
+	// DisplayName is the display name of the profile
+	DisplayName string `json:"display_name" db:"display_name"`
+	// Identifier is the identifier of the profile
+	Identifier string `json:"identifier" db:"identifier"`
+	// InstallDate is the date the profile was installed
```

Comment:
Nit, just to make it clear that this timestamp is from a different source, I'd add `... date the profile was installed on the host as reported by the host's clock.` or something like that.